crosshair/libimpl/builtinslib.py

```diff
@@ -2014,8 +2020,7 @@ def make_registrations():
             'splitlines',
             'startswith',
             'strip',
-            'translate',
-            'zfill',
```

Comment:
OOC, how did you end up with this edit? I haven't explained this section yet, but it exists to handle the case where a **native** python string calls `zfill` on a **symbolic** argument (SmtInt). (we do this by monkey-patching the built-in str class!!)

It might be ok to remove, as we've got some fallback tactics to detect this kind of case, but if you don't need this change, I'd defer it for another CL (or wait for me to pick it up!)

Reply:
Ooh, interesting. I came up with that change simply because I did a CTRL+F for `zfill` and assumed this section of code was providing a fallback for `zfill` **in general**, not just for the case you outline (native string calls zfill with SMTInt).

In any case, I'll take a mental note to remove this change and to investigate this part of the code in the future!